internal/pkg/policy/self.go

```diff
@@ -187,6 +222,49 @@ func (m *selfMonitorT) groupByLatest(policies []model.Policy) map[string]model.P
 	return groupByLatest(policies)
 }
 
+// getPolicyOutput will return the Elasticsearch output block of m.policy if there is a new revision.
+func (m *selfMonitorT) getPolicyOutput() (config.Elasticsearch, error) {
+	var policyES config.Elasticsearch
+	// policy revision has not changed
+	if m.policy.RevisionIdx == m.lastRev {
+		return policyES, ErrNoPolicyUpdate
+	}
+	// always copy revisionIdx
+	m.lastRev = m.policy.RevisionIdx
+
+	// Find elasticsearch output in the policy
+	// TODO figure out how to get output name from policy in order not to scan outputs?
+	for name, data := range m.policy.Data.Outputs {
```

Comment:
so the logic takes the first elasticsearch output, right? If there are more than one elasticsearch outputs, and the first one is not accessible, fleet-server will never try the second one, is that right?

Reply:
correct, i want to be able to use the (output) name associated with the integration directly but i'm not sure where I can find the value